### ISSUE TITLE
Add a text only mode

### ISSUE THREAD

**Issue by clochix** [cozy/cozy-emails#564]
Some users would like to use PGP with our Mail app. Unfortunatly, encrypting / signing / verifying an HTML message doesn't really make sense. So a first step to allow users to use PGP would be to allow to compose messages in plain text / display only the text content.

**Comment by frankrousseau** [comment]
Yep, a little bit more work and we'll be able to put settings back.
